assets/berkeley.yaml

```diff
@@ -0,0 +1,72 @@
+---
+
+- type: model
+  name: Koala
+  organization: Berkeley
+  description: A relatively small chatbot trained by fine-tuning Meta’s LLaMA on
+    dialogue data gathered from the web.
+  created_date: 2023-04-03
+  url: https://bair.berkeley.edu/blog/2023/04/03/koala/
+  model_card: https://huggingface.co/TheBloke/koala-7B-GPTQ-4bit-128g
+  modality: natural language text
+  analysis: Evaluated in comparison with ChatGPT and Stanford Alpaca.
+  size: 13B parameters
+  dependencies: [LLaMA, web-scraped dialogue data]
+  training_emissions: ''
+  training_time: 6 hours
+  training_hardware: 8 A100 GPUs
+  quality_control: ''
+  access: open
+  license: Apache 2.0
+  intended_uses: academic research
+  prohibited_uses: ''
+  monitoring: ''
+  feedback: https://huggingface.co/TheBloke/koala-7B-GPTQ-4bit-128g/discussions
+
+- type: model
+  name: Gorilla
+  organization: Berkeley
+  description: Gorilla is a finetuned LLaMA-based model that surpasses the performance
+    of GPT-4 on writing API calls.
+  created_date: 2023-05-24
+  url: https://arxiv.org/pdf/2305.15334v1.pdf
+  model_card: ''
+  modality: outputs API from natural language input
+  analysis: Evaluated using AST sub-tree matching technique and compared to other
+    models in terms of API functionality accuracy.
+  size: 7B parameters
+  dependencies: [LLaMA, Gorilla document retriever]
+  training_emissions: ''
+  training_time: ''
+  training_hardware: ''
+  quality_control: No specific quality control is mentioned in model training, though
+    details on data processing and collection are provided in the paper.
+  access: open
+  license: Apache 2.0
+  intended_uses: In conjunction with a LLM to improve its capability for using API
+    calls.
+  prohibited_uses: ''
+  monitoring: ''
+  feedback: ''
+
+- type: model
+  name: OpenLLaMA
+  organization: Berkeley
+  description: OpenLlama is an open source reproduction of Meta's LLaMA model.
+  created_date: 2023-05-03
+  url: https://github.com/openlm-research/open_llama
+  model_card: ''
+  modality: text
+  analysis: Evaluated on wide range of tasks using own evaluation benchmarks.
+  size: 17B parameters
+  dependencies: [LLaMA, RedPajama]
```

Comment:
I don't think this depends on LLaMA? If they are reproducing LLaMA, then it probably cannot depend on LLaMA?